pootle/apps/pootle_store/store/serialize.py

```diff
@@ -50,8 +50,9 @@ def tostring(self):
         if hasattr(store, "updateheader"):
             # FIXME We need those headers on import
             # However some formats just don't support setting metadata
+            max_unit_revision = self.max_unit_revision or 0
             store.updateheader(add=True, X_Pootle_Path=self.pootle_path)
-            store.updateheader(add=True, X_Pootle_Revision=self.max_unit_revision)
```

Comment:
From semantic PoV max_unit_revision returns `None` and I think it's a right thing. If we change it then we need to check if it changes anything else (I preferred to avoid thinking about it).